rspace/src/main/scala/coop/rchain/rspace/history/package.scala

```diff
@@ -34,10 +34,17 @@ package object history {
 
   private val logger: Logger = Logger[this.type]
 
+  @deprecated("Use `history.initalize(branch)` instead.", "0.4.2")
```

Comment:
I don't want to use `@deprecated` annotations.  

We have no obligation to guarantee API stability and as such it just clutters the code.

Reply:
So just delete....

Reply:
Yeah, I used the compiler warnings to fix the code, so this is just a leftover.